tests/unit/workflows/test_workflows_tasks_submission.py

```diff
@@ -33,6 +33,7 @@
     create_ticket,
     filter_keywords,
     prepare_keywords,
+    remove_references,
```

Comment:
why?

Reply:
oops... pushed also the revert xd, one sec